src/Analysis/GWASV2/Shared/SourceMock.jsx

```diff
@@ -0,0 +1,37 @@
+import React, {
+  useContext, useState, useEffect, createContext,
+} from 'react';
+import PropTypes from 'prop-types';
+
+// ======== MOCK SourceContext for storybooks and tests ================ //
+
+const SourceMockContext = createContext();
+
+export function SourceMockContextProvider({ children }) {
+  const [source, setSource] = useState(undefined);
+  const sourceId = 123; // hard-coded MOCK value instead of fetching from cohort-middleware
+  useEffect(() => {
+    setSource(sourceId);
```

Comment:
Could we just set the state to have a default value of 123 at line 11? Or am I missing something?

Reply:
Never mind, no longer relevant.